src/pages/[platform]/build-a-backend/auth/manage-users/manage-webauthn-credentials/index.mdx

```diff
@@ -28,3 +28,111 @@ export function getStaticProps() {
   };
 }
 
+<InlineFilter filters={["react-native"]}>
+<Callout warning>
+
+Passwordless features are currently not supported on react native.
```

Comment:
Something more like below would be better. Maybe worth mentioning that passkey lifecycle management APIs are available.

```suggestion
WebAuthn registration and authentication are not currently supported on React Native, other passwordless features are fully supported.
```